### PR TITLE
feat: expand doc images on click 

### LINKED ISSUE #26478: [website]: expand docs images on click
Can we make images expand to fill the screen on-click? Right now images such as this one:

https://geth.ethereum.org/images/docs/grafana.png

are hard to see in their embedded size.

### PR DESCRIPTION
**Pull request by nhsz**
This PR makes images clickable and open full size version on new tab.

**Preview:** https://deploy-preview-26680--geth-website.netlify.app/docs/developers/dapp-developer/dev-mode#compile-and-deploy

## Related issues

Fixes #26478

### CODE REVIEW

File: src/components/UI/docs/MDComponents.tsx [hunk 1]
```diff
@@ -14,12 +16,27 @@ import NextLink from 'next/link';
...
 import { parseHeadingId } from '../../../utils/parseHeadingId';
+import { childrenIsAnImage } from '../../../utils';
 
 const { h1, h2, h3, h4 } = textStyles;
 
 const MDComponents = {
-  // paragraphs
+  // paragraphs & images
   p: ({ children }: any) => {
+    // images in markdown are rendered as children of p tags for some reason
```
Comment (wackerow) [inline src/components/UI/docs/MDComponents.tsx:26]: 😆

File: src/utils/childrenIsAnImage.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const childrenIsAnImage = (children: any) => {
+  return typeof children[0] === 'object' && children[0].props.hasOwnProperty('src');
```
Comment (wackerow) [inline src/utils/childrenIsAnImage.ts:2]: Just thinking, would there be anything else that may contain a `src` property that could cause issues here? I can't think of anything from markdown, just making sure.

**Review by wackerow — APPROVED** [review]
LGTM, nice job @nhsz!

File: src/utils/childrenIsAnImage.ts (hunk 2 above)
Comment (nhsz) [reply src/utils/childrenIsAnImage.ts:2]: Don't think so, we have embedded videos on the docs which are also children of p tags, but isntead of `src` they have `href`.

**Pull request merged**